src_python/habitat_sim/utils/datasets_download.py

```diff
@@ -67,6 +67,42 @@ def initialize_test_data_sources(data_path):
         #   "link": symlink to the data directory pointing to the active version directory
         #   "version": data version tag
         # }
+        "hssd-hab": {
+            "source": "https://huggingface.co/datasets/hssd/hssd-hab.git",
+            "link": data_path + "scene_datasets/hssd-hab",
+            "version": "v0.2.3",
+            "requires_auth": True,
+        },
+        "hssd-raw": {
+            "source": "https://huggingface.co/datasets/hssd/hssd-scenes.git",
+            "link": data_path + "scene_datasets/hssd-scenes",
+            "version": "main",
+            "requires_auth": False,
```

Comment:
These repos are "gated", that means auth is required, right?

Reply:
You are correct. Fixed it!